helix-term/src/commands/typed.rs

```diff
@@ -1656,13 +1656,20 @@ fn language(
         return Ok(());
     }
 
+    if args.is_empty() {
+        let doc = doc!(cx.editor);
+        let language = &doc.language_name().unwrap_or(DEFAULT_LANGUAGE);
+        cx.editor.set_status(language.to_string());
+        return Ok(());
+    }
+
     if args.len() != 1 {
         anyhow::bail!("Bad arguments. Usage: `:set-language language`");
     }
 
     let doc = doc_mut!(cx.editor);
 
-    if args[0] == "text" {
+    if args[0] == DEFAULT_LANGUAGE {
```

Comment:
small style nit but I think we should call this `DEFAULT_LANGUAGE_NAME`. "Language" is already used in core either to refer to tree_sitter::Language or to language configuration